forge/db/views/DeviceGroup.js

```diff
@@ -0,0 +1,63 @@
+module.exports = function (app) {
+    app.addSchema({
+        $id: 'DeviceGroupSummary',
+        type: 'object',
+        properties: {
```

Comment:
This schema doesn't match what I can see the `deviceGroupSummary` function returning. Note sure what `model` relates to.